libraries/chain/chain_config.cpp

```diff
@@ -39,17 +39,53 @@ namespace eosio { namespace chain {
                   "max authority depth should be at least 1" );
 }
 
+template<typename Stream>
+Stream& chain_config_v0::log(Stream& out) const{
```

Comment:
Is this used anywhere?  If it's defined here it won't be instantiated if it's used in another translation unit.